addon-test-support/ember-qunit/-internal/test-debug-info.js

```diff
@@ -47,7 +48,12 @@ export default class TestDebugInfo {
 
   get summary() {
     if (!this._summaryInfo) {
-      this._summaryInfo = Object.assign({ fullTestName: this.fullTestName }, this.settledState);
+      this._summaryInfo = Object.assign(
```

Comment:
I think we will have to use `Ember.assign` instead of `Object.assign`, since we still need to support IE11...

Reply:
Fixed.